.github/workflows/docker-lint.yml

```diff
@@ -27,28 +27,24 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v3
 
-      - name: Install npm and dockerfilelint
+      - name: Install hadolint
         run: |
-          sudo apt-get update
-          sudo apt-get install nodejs npm -y --no-install-recommends
-          npm install -g dockerfilelint
-          wget https://github.com/replicatedhq/dockerfilelint/pull/201.patch -O /usr/local/lib/node_modules/dockerfilelint/201.patch
-          CURRENT_DIR=$PWD
-          cd /usr/local/lib/node_modules/dockerfilelint/
-          git apply 201.patch
-          cd $CURRENT_DIR
-          cat << RULES > ./.dockerfilelintrc
-          rules:
-            sudo_usage: off
-          RULES
+          sudo wget https://github.com/hadolint/hadolint/releases/download/v2.12.0/hadolint-Linux-x86_64 -O /usr/bin/hadolint
```

Comment:
should always get the latest version automatically like in update-helm.sh

Reply:
https://github.com/hadolint/hadolint/releases/latest/download/hadolint-Linux-x86_64

Reply:
did you seen that?

Reply:
Yed, thanks! :)

Reply:
```suggestion
          sudo wget https://github.com/hadolint/hadolint/releases/latest/download/hadolint-Linux-x86_64 -O /usr/bin/hadolint
```